build/jobs/provision-deploy.yml

```diff
@@ -22,6 +22,9 @@ parameters:
 - name: schemaAutomaticUpdatesEnabled
   type: boolean
   default: false
+- name: reindexEnabled
+  type: boolean
+  default: false
```

Comment:
I don't think this is needed as a parameter. Is there a case where we would want to deploy to our test servers without reindex enabled?